module/VuFind/config/module.config.php

```diff
@@ -346,6 +346,7 @@
             'VuFind\Solr\Writer' => 'VuFind\Solr\WriterFactory',
             'VuFind\Tags' => 'VuFind\TagsFactory',
             'VuFind\Connection\WorldCatUtils' => 'VuFind\Connection\WorldCatUtilsFactory',
+            'VuFind\Validator\Csrf' => 'VuFind\Service\Factory::getCsrfValidator',
```

Comment:
Would it make sense to name this service for the original Zend class and avoid the VuFind subclassing entirely? You could still override it locally if necessary.